tests/index.js

```diff
@@ -770,6 +762,7 @@ async function testFindChangedTargets() {
         '//b:library',
         '//b:lint',
         '//b:test',
+        '//d:library',
```

Comment:
2) d:library should be listed here because changes.txt lists non-js/x.txt as a changed file

Reply:
maybe add this context as a comment in the test itself?